src/App.tsx

```diff
@@ -7,8 +7,32 @@ import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { getTodos } from './api';
+import { actions as todosActions } from './features/todos';
 
 export const App: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const [isLoading, setIsLoading] = React.useState(false);
+
+  const loadTodos = async () => {
+    setIsLoading(true);
+    try {
+      const todosFromServer = await getTodos();
+
+      dispatch(todosActions.setTodos(todosFromServer));
+    } catch {
+      Promise.reject();
```

Comment:
what is it for?)) I suggest to set error in the state and show it somewhere on UI